doc/decisions/0_drafts/errors_plugins.md

```diff
@@ -0,0 +1,103 @@
+# Error Semantics
+
+## Problem
+
+While we have a classification of errors and warnings, it remains unclear when plugins actually should emit errors and warnings.
+
+## Constraints
+
+- Should not be contradicting to specified behavior in [storage plugin tutorial](/doc/tutorials/storage-plugins.md).
+
+## Assumptions
+
+- Users want a uniform behavior within Elektra, so plugins must behave uniformly.
+
+## Considered Alternatives
+
+### Free-for-all
+
+Every plugin author decides for themselves.
+
+Leads to inconsistent behaviour and hard to diagnose interaction issues.
+
+### Detailed ruleset with conformance tests
+
+Write a detailed formal ruleset and generic conformance tests for all plugins.
+
+Defining the ruleset, let alone writing the tests, for all possible errors in plugins is borderline impossible.
+
+### Simple guidelines
```

Comment:
I fully agree with this decision. You can directly place the guidelines to `doc/contrib` in the next decision round.

Reply:
With "next round" do you mean the next PR, once this one is merged, or still within this PR?

Reply:
Hard to say. Not so ideal that these two decisions are in one PR.

Reply:
> Not so ideal that these two decisions are in one PR.

I didn't expect this much discussion... I thought the decisions are both very straightforward.